mscore/editstyle.cpp

```diff
@@ -735,6 +735,7 @@ const std::map<ElementType, EditStylePage> EditStyle::PAGES = {
       { ElementType::FIGURED_BASS,        &EditStyle::PageFiguredBass },
       { ElementType::HARMONY,             &EditStyle::PageChordSymbols },
       { ElementType::FRET_DIAGRAM,        &EditStyle::PageFretboardDiagrams },
+      { ElementType::TEXT,                &EditStyle::PageHeaderFooter }, // gotoElement if the element is Text leads to the footer dialog
```

Comment:
Wouldn't this also catch other text elements?

Reply:
It will, as I explained in the original message.

Reply:
Just to be clear, this adds "Style..." to the context menu for *every text element*, and all of them go to the Header/Footer page.

Reply:
You are right, I hadn't seen that connection. I'll change this then.

Reply:
@IsaacWeiss I can think of 2 solutions. Either through 'friend' (like the PR mentioned in the op) or by hardcoding the value of the page and having a unit test to check if the value is correct.